docs/source/main_classes/model.rst

```diff
@@ -41,3 +41,9 @@ The other methods that are common to each model are defined in :class:`~transfor
 
 .. autoclass:: transformers.modeling_tf_utils.TFModelUtilsMixin
     :members:
+
+
+Generative models
+~~~~~~~~~~~~~~~~~
+
+Coming soon
```

Comment:
Will be my next PR doc, but I needed the anchor to make a link in several docstrings of Pipeline subclasses.